client/src/sections/courses/LearningSection/index.tsx

```diff
@@ -90,12 +92,13 @@ const LearningSection: React.FC = () => {
         {learners && learners.length > 0 ? (
           <div className="mx-4 mb-4">
             {/* SORT BUTTON  */}
-            <div className="flex text-[15px] my-2 cursor-pointer">
-              <Dropdown
+            <div className="flex text-[15px] my-2 cursor-pointer justify-end">
```

Comment:
If I remember correctly, I think the filter button is supposed to be on the left (based on course detail page design).